docs/specifications/tests/Basic-TP/basic02.md

```diff
@@ -1,45 +1,76 @@
-## BASIC02: The domain must have at least one working name server
+# BASIC02: The domain must have at least one working name server
 
-### Test case identifier
+## Test case identifier
 **BASIC02** The domain must have at least one working name server
 
-### Objective
+## Objective
 
 In order for the domain to work, it must have at least one name server that
 can answer queries about the domain. 
 
-### Inputs
+## Inputs
+
+* The label of the domain name to be tested ("child zone").
+* If undelegated test, the list of name servers ("undelegated NS").
```

Comment:
I suggest clarifying that this list represents the delegation for the child zone.